keyboards/cannonkeys/satisfaction75/keymaps/boy_314/keymap.c

```diff
@@ -16,6 +16,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
 #include QMK_KEYBOARD_H
+#include "satisfaction_keycodes.h"
```

Comment:
This isn't a great way to handle this. 
If anything, it should be added to the keyboard's header file, rather than to each and every keyboard.

Reply:
You're right - that is a much better solution! I've changed the firmware to take this into account.